frontend/src/components/UserProfileCard.vue

```diff
@@ -17,10 +17,7 @@
         </template>
         <template #content>
           <div v-if="isSafe">
-            <UserStatus
-              :socketCount="user?.socketCount"
-              :gameStatus="user?.gameStatus"
-            />
+            <UserStatus :userId="user?.id" :gameStatus="user?.gameStatus" />
```

Comment:
I can see the status on ChatBoxUserProfileDialogue but it's not shown further on the profile view of the other user (if I click View Profile). It's shown if I click this button on the ChatBoxUserProfileDialogue for my own user